src/c++/perf_analyzer/load_manager.h

```diff
@@ -332,6 +336,8 @@ class LoadManager {
   std::mutex wake_mutex_;
 
 #ifndef DOCTEST_CONFIG_DISABLE
+  friend TestRequestRateManager;
```

Comment:
This is also technically not needed right now as all members that the test class uses are protected or public.

Reply:
removed